engine/lib/views.php

```diff
@@ -1602,12 +1602,12 @@ function elgg_views_boot() {
 	elgg_register_simplecache_view('js/sprintf.js');
 	elgg_register_simplecache_view('js/text.js');
 
-	elgg_register_js('elgg.require_config', elgg_get_simplecache_url('js', 'elgg/require_config'), 'head');
-	elgg_register_js('require', elgg_get_simplecache_url('js', 'require.js'), 'head');
-	elgg_register_js('jquery', elgg_get_simplecache_url('js', 'jquery.js'), 'head');
-	elgg_register_js('jquery-migrate', elgg_get_simplecache_url('js', 'jquery-migrate.js'), 'head');
-	elgg_register_js('jquery-ui', elgg_get_simplecache_url('js', 'jquery.ui.js'), 'head');
-	elgg_register_js('elgg', elgg_get_simplecache_url('js', 'elgg'), 'head');
+	elgg_register_js('elgg.require_config', elgg_get_simplecache_url('js/elgg/require_config'), 'head');
+	elgg_register_js('require', elgg_get_simplecache_url('js/require.js'), 'head');
```

Comment:
For each of these `elgg_get_simplecache_url` we can drop the associated `elgg_register_simplecache_view` call above.